csuntechlab/iSTART
# STR-240 -- FE: Add functionality to allow card-flips to be flipped and un-flipped at least once to continue.

# Description
Redid check for card-flip to allow clicking the card once for all cards to continue.
  
# Testing
1. Ensure flipping all cards enables continue button.
2. Refresh the page and check that flipping the last card *only* doesn't allow you to continue.
3. Un-flip the last card, and flip the other cards, ensure you are able to continue.
4. Refresh the page and flip & un-flip each card.
